src/components/ChangePlan/Checkout/PurchaseSummary/PurchaseSummary.js

```diff
@@ -186,25 +168,61 @@ export const Promocode = () => {
   );
 };
 
-export const TotalPurchase = ({ price, discountPrepayment, discountPaymentAlreadyPaid }) => {
+export const InvoiceInformation = ({ priceToPay }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  priceToPay = 0;
```

Comment:
Is this ok? To set the priceToPay directly to zero. 🤔

Reply:
it's weird

Reply:
Sorry @cbernat, was my mistake, I tested the scenario when the priceToPay is Zero and I forgot remove. I will remove this line.

Reply:
Done